src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -1056,34 +1015,6 @@ public void LoadGeneric4(InputType inputType, ReaderType readerType)
             Assert.True(false);
         }
 
-        //[Variation("Call Load with null custom resolver and style sheet has import/include, should fail")]
-        [InlineData(InputType.Reader, ReaderType.XmlValidatingReader)]
-        [InlineData(InputType.URI, ReaderType.XmlValidatingReader)]
-        [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader)]
-        [Theory(Skip = "By design bug #84957: Skip this test for Load(url, resolver)")]
-        public void LoadGeneric5(InputType inputType, ReaderType readerType)
-        {
-            // By design bug #84957: Skip this test for Load(url, resolver)
-            //if (MyInputType() == InputType.URI)
-            //{
-            //    _output.WriteLine("By design bug #84957: Skip this test for Load(url, resolver)");
-            //    return TEST_SKIPPED;
-            //}
-
-            CustomNullResolver myResolver = new CustomNullResolver(null);
```

Comment:
It became redundant with `TC_CustomNullResolver_Then_XmlUrlResolver`, this one was to be `TC_CustomNullResolver` but it was going to be just a repeat of the first part of the former.